game.py

```diff
@@ -1,52 +1,23 @@
+import random
+from  wonderwords import RandomWord
+#https://pypi.org/project/wonderwords/
+
+SNOWMAN_GRAPHIC = [
+    '*   *   *  ',
+    ' *   _ *   ',
+    '   _[_]_ * ',
+    '  * (")    ',
+    '  \\( : )/ *',
+    '* (_ : _)  ',
+    '-----------'
+]
+
+SNOWMAN_WRONG_GUESSES = len(SNOWMAN_GRAPHIC)
 SNOWMAN_MIN_WORD_LENGTH = 5
 SNOWMAN_MAX_WORD_LENGTH = 8
 SNOWMAN_MAX_WRONG_GUESSES = 7
 
-SNOWMAN_1 = '*   *   *  '
-SNOWMAN_2 = ' *   _ *   '
-SNOWMAN_3 = '   _[_]_ * '
-SNOWMAN_4 = '  * (")    '
-SNOWMAN_5 = '  \\( : )/ *'
-SNOWMAN_6 = '* (_ : _)  '
-SNOWMAN_7 = '-----------'
-
-def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
```

Comment:
We could minimize the changes in the file by adding the `snowman` implementation here, rather than moving it later in the file.